packages/sdk/src/index.ts

```diff
@@ -31,8 +31,13 @@ export {
   type RealTimeModels,
   type VideoModelDefinition,
   type VideoModels,
+  realtimeModels,
+  videoModels,
+  imageModels,
+  modelSchema,
+  modelInputSchemas,
```

Comment:
Do we want to expose this for the `isRealtime`/`isVideo`/`isImage` checks?

Exposing the Zod schemas directly would make Zod - and these specific schemas - part of our public API, which I'm not sure is ideal. It would only benefit our Zod-based consumers, and limit our ability to evolve the schemas freely or, if needed, change Zod in favor of something else in the future.

What do you think? Maybe we should expose more targeted, explicit checks instead?